frontend/src/components/About/About.tsx

```diff
@@ -1,19 +1,28 @@
 import React from 'react'
 import { Button, MainButton, SecondaryButton } from '../toolbox/Buttons'
+import { AppBar } from '../toolbox/AppBar';
 import * as s from './About.theme';
+import { LOCALHOST_ABOUT_PAGE, LOCALHOST_HOME_PAGE, HOME_TITLE, ABOUT_TITLE } from '../../constants/links';
 
 export const About = () => {
 
    const renderArtistsBio = () => {
       return (
+         <>
          <s.ArtistsBio>
             <p>Hi My Name is Lucy</p>
          </s.ArtistsBio>
-         
+         </>
       )
    }
 
    return (<>
+      <AppBar
```

Comment:
Instead of placing this on every single page, how about wrapping it on App.tsx so it gets applied to every page universally?